common/persistence/cassandra/util.go

```diff
@@ -1105,6 +1192,18 @@ func resetSignalInfoMap(
 	return sMap, encoding, nil
 }
 
+func resetBlobMap[K comparable](
```

Comment:
Named this way because it serves the same purpose as resetTimerInfoMap, resetSignalInfoMap, and resetActivityInfoMap that already exist. But this one is generic over the map key type.